### PR TITLE
Fixing cloud auth

### PR DESCRIPTION
**Pull request by atoonk**
Deprecating cloud auth in favor of policies
Printing warning if no policies have been applied
Printing Name

### CODE REVIEW

File: cmd/connect.go [hunk 1]
```diff
@@ -212,6 +212,8 @@ func init() {
 	connectCmd.Flags().BoolVarP(&httpserver, "httpserver", "", false, "Start a local http server to accept http connections on this host")
 	connectCmd.Flags().StringVarP(&httpserver_dir, "httpserver_dir", "", "", "Directory to serve http connections on this host")
 	connectCmd.Flags().MarkDeprecated("localssh", "use --sshserver instead")
+	connectCmd.Flags().MarkDeprecated("allowed_email_domains", "use policies instead")
```
Comment (btoonk) [inline cmd/connect.go:215]: The depreciation only works for the long name, not the short flag (-e -d), maybe use MarkShorthandDeprecated or both.

File: cmd/root.go [hunk 2]
```diff
@@ -5,7 +5,7 @@ Licensed under the Apache License, Version 2.0 (the "License");
 you may not use this file except in compliance with the License.
 You may obtain a copy of the License at
 
-    http://www.apache.org/licenses/LICENSE-2.0
+	http://www.apache.org/licenses/LICENSE-2.0
```
Comment (btoonk) [inline cmd/root.go:8]: Is this an intended change?

File: cmd/socket.go [hunk 3]
```diff
@@ -404,6 +404,8 @@ func init() {
 	socketConnectCmd.Flags().BoolVarP(&localssh, "localssh", "", false, "Start a local SSH server to accept SSH sessions on this host")
 	socketConnectCmd.Flags().BoolVarP(&localssh, "sshserver", "l", false, "Start a local SSH server to accept SSH sessions on this host")
 	socketConnectCmd.Flags().MarkDeprecated("localssh", "use --sshserver instead")
+	socketCreateCmd.Flags().MarkDeprecated("allowed_email_domains", "use policies instead")
```
Comment (btoonk) [inline cmd/socket.go:407]: can you move these up to the other socketCreate flags?

File: cmd/root.go (hunk 2 above)
Comment (atoonk) [reply cmd/root.go:8]: Looks like VScode. made it a real tab

File: cmd/socket.go (hunk 3 above)
Comment (atoonk) [reply cmd/socket.go]: Yup sone

File: cmd/root.go (hunk 2 above)
Comment (atoonk) [reply cmd/root.go:8]: VScode made it a real tab :)

**Review by btoonk — APPROVED** [review]

**Pull request merged**